index.html

```diff
@@ -0,0 +1,250 @@
+<html>
+
+<head>
+    <title>Project Kmarketer</title>
+    <link src="css/style.css" type="text/css">
+
+
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+        body {
```

Comment:
since you already have style.css file. Would be logic to move all your CSS code there from your index.html